src/main/java/org/apache/mesos/hdfs/HDFSScheduler.java

```diff
@@ -236,16 +251,14 @@ public void run() {
       }
     } catch (InterruptedException | ExecutionException | InvalidProtocolBufferException e) {
       log.error("Error recovering framework id", e);
-      throw new RuntimeException(e);
     }
 
-    MesosSchedulerDriver driver = new MesosSchedulerDriver(this, frameworkInfo.build(),
-        conf.getMesosMasterUri());
+    MesosSchedulerDriver driver = new MesosSchedulerDriver(this, frameworkInfo.build(), conf.getMesosMasterUri());
     driver.run();
   }
 
   private void launchNode(SchedulerDriver driver, Offer offer,
-        String nodeName, List<String> taskNames, String executorName) {
+                          String nodeName, List<String> taskNames, String executorName) {
```

Comment:
I'm not terribly picky myself, as long as we stick to some convention.

As a point of reference (which is by no means a recommendation), in Mesos, either all parameters fit on one 80-char line, or they each get their own line. If the first parameter starts too many columns out already (arbitrary judgement), then it is wrapped to the next line and they all start at one indentation level in. It seems fairly readable, but makes for a fuzzier auto-formatting rule.